app/src/main/AndroidManifest.xml

```diff
@@ -2,7 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.lhi06.residue">
 
-    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
this is why i had them dependencies in as i had coded the API function thing but dalli reverted it and took it out until i get back to that ticket